app/src/main/java/com/mobileproto/hireddit/hireddit/SpeechListener.java

```diff
@@ -0,0 +1,82 @@
+package com.mobileproto.hireddit.hireddit;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.speech.RecognitionListener;
+import android.speech.SpeechRecognizer;
+import android.util.Log;
+import android.view.View;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+/**
+ * Voice Recognition: For speech to text input
+ */
+
+class SpeechListener implements RecognitionListener {
+    String TAG = "myDebug";
```

Comment:
i did have it like that before but i couldn't find a way to filter
properly. ill change it though

On Wed, Nov 11, 2015 at 11:15 AM, greneroom <notifications@github.com>
wrote:

> In app/src/main/java/com/mobileproto/hireddit/hireddit/SpeechListener.java
> <https://github.com/greneroom/Hi-Reddit/pull/7#discussion_r44550405>:
>
> > +import android.app.Fragment;
> > +import android.os.Bundle;
> > +import android.speech.RecognitionListener;
> > +import android.speech.SpeechRecognizer;
> > +import android.util.Log;
> > +import android.view.View;
> > +import android.widget.Toast;
> > +
> > +import java.util.ArrayList;
> > +
> > +/**
> > + * Voice Recognition: For speech to text input
> > + */
> > +
> > +class SpeechListener implements RecognitionListener {
> > +    String TAG = "myDebug";
>
> I meant that maybe you should make the tag itself more specific, so if
> there's a debug statement in another Fragment you can tell the difference.
> Ie,
>
> String SPEECH_TAG = "speechDebug"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/greneroom/Hi-Reddit/pull/7/files#r44550405>.
>